packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -149,14 +149,7 @@ export function isConformant(
           // silence element nesting warnings
           consoleUtil.disableOnce();
 
-          const wrapper = mount(
-            <Component
-              {...{
-                ...(defaultAs && { as: defaultAs }),
```

Comment:
This is non-sense: we pass `as` and assert against it, while we should be testing for a default value.